src/app/modules/choice-menu/choice-menu.component.ts

```diff
@@ -12,6 +12,11 @@ export interface ActionDef {
   templateUrl: './choice-menu.component.html',
   styleUrls: ['./choice-menu.component.scss']
 })
+
+export interface ActionOptDef {
+  gridCols: number;
```

Comment:
Upon the comments in the files below, there seems no need to create additional interface for the defined properties on the `ChoiceMenuComponent` itself.